pyuntl/untldoc.py

```diff
@@ -242,7 +240,7 @@ def post2pydict(post, ignore_list):
             elif len(child_list) > 0:
                 untl_element = PYUNTL_DISPATCH[element_tag]()
             # If the element has children, add them.
-            if len(child_list) > 0 and untl_element is not None:
+            if child_list and untl_element is not None:
```

Comment:
We could probably just simplify this to: 
```suggestion
            if child_list and untl_element:
```